airflow/jobs/scheduler_job.py

```diff
@@ -26,7 +26,7 @@
 import time
 from collections import defaultdict
 from contextlib import redirect_stderr, redirect_stdout, suppress
-from datetime import timedelta
+from datetime import datetime, timedelta
```

Comment:
```suggestion
from datetime import timedelta
```

(Unused import error from tests.)